__tests__/GraphBuilder.test.js

```diff
@@ -38,6 +38,44 @@ describe("GraphBuilder", () => {
       expect(new GraphBuilder(resource).graph.toCanonical()).toMatch(rdf)
     })
 
+    it("builds a graph for literal with validationDataType", () => {
+      const resource = build.subject({
+        subjectTemplate: build.subjectTemplate({
+          id: "resourceTemplate:testing:literalValidation",
+          clazz: "http://sinopia.io/testing/LiteralValidation",
+        }),
+        properties: [
+          build.literalProperty({
+            propertyTemplate: build.propertyTemplate({
+              subjectTemplateKey: "resourceTemplate:testing:literalValidation",
+              label: "literalValidation, integer validationDataType",
+              uris: {
+                "http://sinopia.io/testing/LiteralValidation/property3":
+                  "http://sinopia.io/testing/LiteralValidation/property3",
+              },
+              type: "literal",
+              validationDataType: "http://www.w3.org/2001/XMLSchema/integer",
```

Comment:
Turns out I had this wrong. It should be `http://www.w3.org/2001/XMLSchema#integer` (and same for other datatypes).

Reply:
@peetucket this relates to your PR as well.   I guess your PR now needs to depend on this PR, which will contain the fix.

Reply:
yes, that request was also noted in my PR, and I fixed it and then changed it everywhere else in the code that it occurred: https://github.com/LD4P/sinopia_editor/pull/3349

that PR was just approved, so we can merge it 

i think this is a new spot though, so you should only need to change this and not the other ones